app/res/layout/setup_keepinstall.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+    <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                    xmlns:SquareButtonWithText="http://schemas.android.com/apk/res/org.commcare.dalvik"
+                    android:id="@+id/screen_first_start_main"
+                    android:layout_width="fill_parent"
+                    android:layout_height="fill_parent"
+                    android:orientation="vertical"
+                    android:background="@color/cc_core_bg"
+        >
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
+            <!--&lt;!&ndash; TODO: extract style definition &ndash;&gt;-->
+            <ImageView
+                android:id="@+id/screen_first_start_banner"
```

Comment:
The second fragment (start/stop install screen) has a few differences (header and footer text) that makes it easier to just have 2 different screens